### DISCUSSION TITLE
Making changes to wire-schema

### DISCUSSION THREAD

**Discussion by snssays** [square/wire#2206] [Q&A]
Hello Team, I was able to parse the .proto file and make changes using ProtoFileElement and generate a new proto file. The problem here is I would like to have my own kind of indentation implementation for it. I have tried to clone it and am trying to make changes to `toSchema()` functions for the services and typeelements. 

However when I am building the project by `./gradlew clean build` . I am running into this 

```

> Configure project :wire-library:wire-runtime-swift
xcrun: error: unable to lookup item 'PlatformPath' from command line tools installation
xcrun: error: unable to lookup item 'PlatformPath' in SDK '/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk'

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':wire-library:wire-runtime-swift'.
> Failed to query the value of task ':wire-library:wire-runtime-swift:compileTestSwift' property 'compilerArgs'.
   > Process 'command 'xcrun'' finished with non-zero exit value 1
```

My changes are only on `wire-schema`. How would I go forward? TIA

P.S I don't have xcode on my machine. I am hoping I won't require that in order to make changes to `wire-schema`.

**Comment by oldergod** [comment]
You can remove the swift related modules from the settings file.

Please, though, we would appreciate if you could ask this kind of questions on [StackOverflow](https://stackoverflow.com/search?q=%5Bsquare-wire%5D&searchOn=3) in the future.
